.github/workflows/ci_tests.yaml

```
@@ -0,0 +1,72 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: PyGMT Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: ${{ matrix.os }} - Python ${{ matrix.python_version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 3
+      fail-fast: false
+      matrix:
+        python_version: [3.7]
```

`matrix.python-version` is used in the setup-miniconda action, not `matrix.python_version`.
```suggestion
        python-version: [3.7]
```

---

Good catch, I'll need to change this for one of my other repos as well!